docker-compose.yml

```diff
@@ -13,5 +13,5 @@ services:
     depends_on:
       - postgres_db
       - mongo_db
-    entrypoint: /app/docker/web/entrypoint.sh
-    privileged: true
+    command: sh "/app/docker/web/entrypoint.sh"
```

Comment:
@gogobook I am curious as to what this change achieves. In command you are executing the same file.

Reply:
it raise an error when I using "entrypoint" way, so I changed it to command.